nix/dapp.nix

```diff
@@ -282,15 +282,15 @@ let
         dss = dss_9746105;
       };
       repo' = {
-        name = "dss-deploy-6a2a00c-source";
+        name = "dss-deploy-20e92b3-source";
         url = "https://github.com/makerdao/dss-deploy";
-        rev = "6a2a00c5c63dbedd97d220040fb46e6885684575";
+        rev = "20e92b322f9612e6968687e367043584f377001a";
         ref = "HEAD";
       };
       src' = fetchGit repo';
       src = "${src'}/src";
     };
-    dss-deploy_d0b018a = rec {
+    dss-deploy_6a2a00c = rec {
```

Comment:
@icetan not sure where this version is coming from and why it was updated.

Reply:
This is coming from the other packages. What I use to do is to update also all the packages that depend on `dss-deploy`. So you need to update `dss-cdp-manager` so then you can update `dss-proxy-actions` and also `line-spell` and `testchain-pause-proxy-actions`

Reply:
Yeah I thought about doing that, but then I wasn't sure that the other contracts actually used dss-deploy to deploy.

`line-spell` for example only uses it in tests (maybe we should think about removing it from tests btw)

Reply:
Oh but you are right, why it was updated to a previous version.

Reply:
Oh btw I realize now it wasn't updated, it was just moved, `6a2a00c` was there already.

Reply:
> Yeah I thought about doing that, but then I wasn't sure that the other contracts actually used dss-deploy to deploy.

Yeah they are only for testing purposes but I like to keep them updated as well so the versions are the same everywhere. But yeah, 100% sure it is not necessary at all, excepting there is a breaking change in `dss-deploy` that comes from `dss`, then you are obligated to update the packages.

> Oh btw I realize now it wasn't updated, it was just moved, `6a2a00c` was there already.

Ahh ok!

Reply:
Yeah ok, let's update them all for now but I [created an issue](https://github.com/makerdao/smart-contracts-team-pm/issues/208) to remove unnecessary deps to `dss-deploy`.